src/bin/deku_cli.ml

```diff
@@ -184,7 +185,7 @@ let info_create_transaction =
   Term.info "create-transaction" ~version:"%\226\128\140%VERSION%%" ~doc ~exits
     ~man
 let create_transaction node_folder sender_wallet_file received_address amount
-    ticket argument vm_flavor =
+    ticket argument vm_flavor tickets =
```

Comment:
Why not merging `ticket` and `tickets` into one single argument and probably add constraints to it later on the code? Having two parameters with almost the same and different purposes makes things a little bit confuse at least.

Reply:
We will be using one ticket to pay for fees, while others are passed in contract argument positions  so we need to track their ownership somehow

Reply:
Right, I think we could make use of more descriptive parameter name.

Reply:
we will change it in June when we will have normal argument parsing for contracts